workflow-templates/conductor-on-main.yaml

```diff
@@ -25,3 +25,4 @@ jobs:
     with:
       env: production
       notify: true
+      automate_irp: true
```

Comment:
@hspearman What does this option do? Automatically freeze deploys?

Reply:
yes, it's what triggers these message: 🧊  »» app «« has been frozen - triggered by Conductor 🤖

Reply:
opt-in by default, with opt-out optional.

Reply:
Got it. Thanks.